### PR TITLE
User Dashboard has been added along with the new mock route for quiz

### PR DESCRIPTION
**Pull request by jay10596**
Hello Jerrell,

I have added the user dashboard which would indicate the user's progress and other courses he would like to sign up for once he is logged in. In addtion, I have also added a new mock route Quiz on the navigation that would display the 10 short Q&A regarding the basics of Laravel.

Thank you.

### CODE REVIEW

**Review by jerrellniu — CHANGES_REQUESTED** [review]
- Don't delete the env.example
- Put images (png/jpg etc) in a img folder in the public directory
- Prefer not to inline PNG's in HTML (we often do with SVGs because we can style in the code)
- text-black is the default colour for text so we shouldn't need to explicitly state it in the HTML

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -15,27 +15,32 @@
...
+                                <a class="no-underline hover:underline text-black text-sm p-3" href="{{ route('register') }}">{{ __('Register') }}</a>
                             @endif
                         @else
-                            <span class="text-gray-300 text-sm pr-4">{{ Auth::user()->name }}</span>
+                            <a class="inline-block py-2 px-4 text-black font-bold no-underline" href="/home">Dashboard</a>
+
+                            <a class="inline-block text-black no-underline hover:text-gray-800 hover:text-underline py-2 px-4" href="/quiz">Quiz</a>
+
+                            
+                            <img class="inline-block rounded-full h-15 w-15 mr-4" src="https://cdn.i-scmp.com/sites/default/files/styles/768x768/public/d8/images/methode/2019/11/04/37716de6-fee6-11e9-93ee-a5388fc1b87d_image_hires_174946.jpg?itok=81dBdlex&v=1572860994">
```
Comment (jerrellniu) [inline resources/views/layouts/app.blade.php:51]: What's the cdn referring to here? Is this an icon? If so we would use an svg say from heroicons.

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -15,27 +15,32 @@
...
                         </a>
```
Comment (stevethomas) [inline resources/views/layouts/app.blade.php]: I suspect this is the Laravel logo?

You should remove this boilerplate.

File: resources/views/welcome.blade.php [hunk 3]
```diff
@@ -32,7 +32,7 @@
         <div class="flex flex-col justify-around h-full">
             <div>
                 <h1 class="text-gray-600 text-center font-light tracking-wider text-5xl mb-6">
-                    {{ config('app.name', 'Laravel') }}
+                    Bootcamp4
```
Comment (stevethomas) [inline resources/views/welcome.blade.php:29]: In `.env` and .`.env.example` you would specific `APP_NAME` as "Bootcamp 4", rather hard code the app name. 

Just revert this change and make it read `config('app.name')`, committing the APP_NAME to `.env.example`.

**Review by stevethomas — CHANGES_REQUESTED** [review]
